### PR TITLE
Adding icons for external cms profile progression module

### PR DESCRIPTION
**Pull request by ericdvb**

### CODE REVIEW

File: src/pineapple.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 17.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```
Comment (hellatan) [inline src/pineapple.svg]: @ericdvb please remove the `xml` and `DOCTYPE` tags from both of these svgs so they can be merged.

File: src/pineapple.svg [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 17.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (hellatan) [inline src/pineapple.svg]: also remove the `id="Layer_1"`

Comment (hellatan) [inline src/pineapple.svg]: `x` and `y` attributes are not needed. Please add the following class attribute: `class="dibsvg-svg dibsvg-<ICON_NAME>"` where `<ICON_NAME>` is either `congratulations` or `pineapple`

**Comment by ericdvb** [comment]
@hellatan updated

**Comment by hellatan** [comment]
published:

> \+ dibs-vg@1.9.0

**Pull request merged**